### ISSUE TITLE
warning: SocketConnector return not used

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/picongpu#2062]
While compiling the current `dev` on taurus I got the following warning:
```
.../picongpu/src/picongpu/include/plugins/output/sockets/SocketConnector.hpp(112): warning: result of call is not used
```

This code has not changed since the alpha release. Do we want to fix this warning? 

System details:
 - cmake/3.3.1
 - cuda/8.0.44
 - gcc/5.3.0
 - bullxmpi/1.2.8.4
 - boost/1.60.0-gnu5.3

**Comment by ax3l** [comment]
I'll push a quick fix: http://kirste.userpage.fu-berlin.de/chemnet/use/info/libc/libc_11.html

**Comment by PrometheusPi** [comment]
solved with #2063
